mailbox/lucene/src/main/java/org/apache/james/mailbox/lucene/search/LuceneMessageSearchIndex.java

```diff
@@ -739,23 +748,32 @@ public void body(BodyDescriptor desc, InputStream in) throws MimeException, IOEx
         MimeStreamParser parser = new MimeStreamParser(config);
         parser.setContentDecoding(true);
         parser.setContentHandler(handler);
-       
+
         try {
             // parse the message to index headers and body
             parser.parse(membership.getFullContent());
         } catch (MimeException e) {
-            // This should never happen as it was parsed before too without problems.            
+            // This should never happen as it was parsed before too without problems.
             throw new MailboxException("Unable to index content of message", e);
         } catch (IOException e) {
             // This should never happen as it was parsed before too without problems.
             // anyway let us just skip the body and headers in the index
             throw new MailboxException("Unable to index content of message", e);
         }
-       
+
 
         return doc;
     }
 
+    private static boolean hasAttachment(MailboxMessage membership) {
+        return membership.getAttachments().stream().anyMatch(new Predicate<MessageAttachment>() {

```

Comment:
Skip more lines (btw stream and anyMatch)

And thanks I did not knew anyMatch ;-)